### ISSUE TITLE
The list of objects for which to solve the problem

### ISSUE THREAD

**Issue by MAlexey** [osTicket/osTicket#2177]
The system simply needs a list of objects for which the problem is solved.
Here is an example: we are engaged in technical support of various equipment. The client can be one machine. and more.
For us it is important to know the model, serial number and year of manufacture. All that the customer must specify each time in the application. It is not convenient for the customer and for us. We can not analyze the problem on a specific machine.
And in the case of entry errors (eg extra space or a point) we have a completely different machine.
Please add to the list of organizations in addition to the contact persons is also an object of the problem.

**Comment by ntozier** [comment]
Go to: Admin panel -> Manage -> Forms ?
or Admin panel -> Manage -> Lists ?

**Comment by MAlexey** [comment]
And? What to do with this list?

**Comment by ntozier** [comment]
And either make your custom field or list and add it to the built in form.

**Comment by infectormp** [comment]
@MAlexey are you try spiceworks? Maybe spiceworks will be better for you?

**Comment by MAlexey** [comment]
Maybe I do not quite understand the use of lists and forms. Unfortunately I did not find description and examples.
In the form of a new application in their ostiket I set up the field, where the customer enters modnl and serial number of the equipment, but I need to have these data were in the same form as the user directory.

spitsevogks does not suit me. ostiket cope with my tasks you want to add quite a few functions.
